pkgs/applications/networking/ids/suricata/default.nix

```diff
@@ -49,6 +49,10 @@ stdenv.mkDerivation rec {
   ++ lib.optionals rustSupport [ rustc cargo ]
   ;
 
+  propagatedBuildInputs = with python.pkgs; [
+    pyyaml
```

Comment:
It doesn't look like this line actually ends up in the final build.
The `suricata` doesn't have any python path set.

Reply:
Never mind. It's `suricatasc`...